README.md

```diff
@@ -1,5 +1,18 @@
 # openweathermap-flutter
 Weather, everybody wants to know how it is going to be during the week. Will it be rainy, windy, or sunny? Luckily for us, in the information age, there are open APIs to retrieve information about it.
 
+## Interfaces
+- **Atmosphere** (day and night)
+- **Clear** (day and night)
+- **Clounds** (day and night)
+- **Drizzle** (day and night)
+- **Rain** (day and night)
+- **Snow** (day and night)
+- **Thunderstorm** (day and night)
```

Comment:
@pedrohsampaioo, I made a change in the other **README.md** (#4 )... I believe that after doing the PR we can update to update both.
Tks